DataStructure/src/main/list/ResizableArray.java

```diff
@@ -83,7 +94,7 @@ public void set(int index, int num)  {
      *  or if the given index is greater than or equals to size.
      */
     public int removeAtIndex(int index) {
-        if (index < 0 || index >= writeIndex) {
+        if (index < 0 || index >= writeIndex || isEmpty()) {
```

Comment:
Creating code review with isEmpty method here = insincerity

Reply:
it was not in my main code I don't know why